apps/web/turbo.json

```diff
@@ -13,7 +13,8 @@
         ".env"
       ],
       "outputs": [".next/**", "!.next/cache/**"],
-      "dependsOn": ["^build"]
+      "dependsOn": ["^build"],
+      "env": ["NEXT_PUBLIC_APP_ENV", "EDGE_CONFIG"]
```

Comment:
Should these affect the build hashes?

Reply:
think we used to have issues with those. not sure why it's in this pr though

Reply:
we need to either disable the lint rule causing all envs to be errors if they aren't in turbo.json or add all the envs in there